spec/rails_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV["RAILS_ENV"] ||= "test"
+ENV["ORGANISATION_FILE_PATH"] = "spec/support/organisation_types.yml"
```

Comment:
Of course, I actually didn't see that there was a default path when fetching the ENV.